engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -136,7 +136,7 @@ class WorkflowActor(workflowId: WorkflowId,
                     serviceRegistryActor: ActorRef)
   extends LoggingFSM[WorkflowActorState, WorkflowActorData] with ActorLogging with KnowsWhatTimeItIs {
 
-  val tag = self.path.name
+  val tag = s"WorkflowActor [UUID(${workflowId.shortString})]"
```

Comment:
Also if the WorkflowActor were created with an appropriate name, this change before and after should actually have been equivalent